docs/layers/base-tile-layer.md

```diff
@@ -81,6 +80,15 @@ The maximum cache size for a tile layer. If not defined, it is calculated using
 
 - Default: `null`
 
+##### `strategy` (Enum, optional)
+
+How the tile layer determines the visibility of tiles. One of the following:
+
+* `'default'`: If some tiles in the current viewport are waiting for data to load, aggresively use cached content from other zoom levels to fill the empty space.
+* `'exclusive'`: Avoid showing overlapping tiles when backfilling with cached content.
+
+- Default: `BaseTileLayer.STRATEGY_DEFAULT`
```

Comment:
The bulleted list has strings. This uses an enum that is a static member of the class, which seems unusual for our API and may require extra wiring to use declaratively.